.travis.yml

```diff
@@ -20,6 +20,7 @@ before_script: &before_script
   - psql -c 'create database travis_ci_test;' -U postgres
   - scripts/no_focus.sh
   - bundle exec rake configure:local
+  - RAILS_ENV=test bundle exec rake db:migrate
```

Comment:
```suggestion
```
this is not required, fwiw https://github.com/zooniverse/panoptes/blob/d0eadcf11d61f4510d92cf73f0c1411104de3ce1/spec/spec_helper.rb#L9
sets up the test database schema from the current `structure.sql`. 

Please make sure any PR migrations have been run locally and the associated changes included in the comitted `structure.sql` file.

Reply:
Running them locally causes the structure.sql diff to balloon with a bunch of noise. I think it's a local PG version issue, so I'm trying to get it done in Docker right now.

Reply:
✔️